inc/namespace.php

```diff
@@ -33,7 +33,6 @@ function set_up_checks() {
 	}
 
 	add_filter( 'wp_insert_post_data', __NAMESPACE__ . '\\block_publish_if_failing', 10, 2 );
-	add_filter( 'rest_pre_insert_post', __NAMESPACE__ . '\\block_publish_for_rest', 10, 2 );
```

Comment:
How does removing this relate to the rest of the newly-added logic?

Reply:
It's moved down into post-init so we can loop over the post types (L172)

Reply:
Aha, thanks, had missed that. Cool!